wgengine/pendopen.go

```diff
@@ -106,14 +125,19 @@ func (e *userspaceEngine) trackOpenPostFilterOut(pp *packet.Parsed, t *tstun.TUN
 
 func (e *userspaceEngine) onOpenTimeout(flow flowtrack.Tuple) {
 	e.mu.Lock()
-	if _, ok := e.pendOpen[flow]; !ok {
+	of, ok := e.pendOpen[flow]
+	if !ok {
 		// Not a tracked flow, or already handled & deleted.
 		e.mu.Unlock()
 		return
 	}
 	delete(e.pendOpen, flow)
 	e.mu.Unlock()
 
+	if of.problem != 0 {
```

Comment:
IsZero or a symbolic name for no-problem, maybe?

Reply:
done